cms/src/test/java/uk/nhs/digital/apispecs/commonmark/ThematicBreakAttributeProviderTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.nhs.digital.apispecs.commonmark;
+
+import static org.mockito.BDDMockito.then;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import org.commonmark.ext.gfm.tables.TableBlock;
+import org.commonmark.node.Node;
+import org.commonmark.node.ThematicBreak;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import java.util.Map;
+
+
+public class ThematicBreakAttributeProviderTest {
+
+    @Mock
+    private Map<String, String> attributes;
+
+    private ThematicBreakAttributeProvider thematicBreakAttributeProvider = new ThematicBreakAttributeProvider();
+
+    @Before
+    public void setUp() {
+        initMocks(this);
+    }
+
+    @Test
+    public void setsCodeInlineCssClassOnCodeElement() {
```

Comment:
The method's name was probably meant to reflect the fact that we're dealing with the 'thematic break' line, here :wink: